METADATA.md

```diff
@@ -1,48 +1,37 @@
 
-  - A user must consent to uploading their data with Metadata.
-  - Metadata associated with something uploaded to IPFS will be permanent, searchable and public and therefore there needs to be consent.
-  - It is best that unconsented Metadata is not appended to Linnia Smart Contracts. 
-  - This functionality should be at App Level and if a user does not consent then data uploaded to Linnia Protocol should be void of Metadata. 
-  - Please take caution, show examples of Metadata and be completely transparent in how a user's data and Metadata are uploaded to the application. It is in the best interest of everyone involved that the privacy of the user is maintained at all times. 
 
-## Metadata Standards
-
-  - Metadata will be specific to domain (i.e Health, Social etc) 
-  - Specific Metadata standards are currently being worked on and investigated. 
-  
-## Metadata and Personally Indentifiable Information (PII)
-
-- The issuance of PII on the Ethereum Blockchain will most definitly be an issue for most people and will be up to the company, developer or application to properly disclose how Metadata for files are being uploaded. Customers will want to make sure their data is secure and that information that they do not want public is not out in the open. 
-- When a person's name is included in the Metadata, someone can ultimately find out the person's Ethereum address through researching the blockchain. This should be kept in mind when uploading Metadata alongside data to Linnia. 
 
 ## Example
 
-```
-PII (Something a customer may not want to uplaoded alongside their data)
-{
-"Name":"Arron Trantow",
-"Test": Glucose,
-"Results: "190 mg/dL"
-}
-```
-```
-Non-PII (A solution that may work)
+```json
 {
-"Customer Number":"7494839843",
-"Test": Glucose,
-"Results: "190 mg/dL"
+    "data_format": "json",
+    "encryption_scheme": "x25519-xsalsa20-poly1305",
+    "encryption_public_key": "hQYhHJpzZH/tGhz1wtqSjkL17tJSnEEC4yVGyNTHNQY=",
+    "linnia_js_version": "0.1.4",
+    "owner_name": "John Linnia",
+    "provider_name": "Facebook",
+    "provider_ethereum_address": "0x349e31e92027f86b0ffeb5cd5e07003c7f229872",
+    "keywords": [ "facebook", "friends list", "people" ],
 }
 ```
 
-```
-Non-PII (A solution stripped of all customer information)
-{
-"Test": Glucose,
-"Results: "190 mg/dL"
-}
-```
 
 
+## Specifications
+
+The Metadata should be in JSON format and the following are the properties that should have:
+
+| property                  | type          | options                     | required |                                                              |
+| ------------------------- | ------------- | --------------------------- | -------- | ------------------------------------------------------------ |
+| data_format               | string        | 'json'   'csv' 'plain_text' | Yes      | Format of the data                                           |
```

Comment:
Camel case instead of snake case

Reply:
ok, and I'll remove the required and explain that this is the recommended metadata specs